src/stories/control/DarkModeButton.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { fireEvent, screen, render } from "@testing-library/react";
+import { DarkModeButton } from "./DarkModeButton";
+import { ColorModeProvider } from "@chakra-ui/react";
+
+describe("scenario", () => {
+
+  it("check if Light Mode", () => {
+    render(<DarkModeButton />)
+    screen.getByLabelText('Light Mode')
+  });
+
+  it("should update label if button clicked", async () => {
+    render(<ColorModeProvider><DarkModeButton /></ColorModeProvider>)
+    fireEvent.click(screen.getByRole("button"))
+    await screen.findByLabelText('Dark Mode')
+    screen.debug()
+  });
+
+  it("check if toggleColorMode called when button clicked", () => {
```

Comment:
Instead of passing it to `onClick`, we should check the behaviour of this component in a way that is as close as possible to the original component. That is, this test might not be necessary